client/src/app/list-contact/list-contact.component.html

```diff
@@ -0,0 +1,13 @@
+<ul class="container" tabindex="0" #list>
+  <li
+    *ngFor="let contact of contacts; let i = index"
+    [ngClass]="{ active: i === contactIndex }"
```

Comment:
```suggestion
    [class.active]="i === contactIndex"
```
https://angular.io/guide/class-binding